File: C++17/백준/Bronze/2566. 최댓값/최댓값.cc
#include<bits/stdc++.h>
using namespace std;
int n,m,mx,a,b;
int arr[103][103];

int main(){
	for(int i=0;i<9;i++){
		for(int j=0;j<9;j++){
			cin>>arr[i][j];
			mx= max(mx,arr[i][j]);
			if(mx==arr[i][j]){
				a = i,b=j;
			}
		}
	}
	cout<<mx<<'\n'<<a+1<<' '<<b+1;
			
}

File: Java/백준/Bronze/2566. 최댓값/최댓값.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException{
        int[][] arr = new int[9][9];
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        for(int i=0;i<9;i++){
            StringTokenizer st = new StringTokenizer(br.readLine());
            for(int j=0;j<9;j++){
                arr[i][j] = Integer.parseInt(st.nextToken());
            }
        }
        int ret = -1;
        int a =0,b=0;
        for(int i=0;i<9;i++){
            for(int j=0;j<9;j++){
                ret =Math.max(ret,arr[i][j]);
                if(ret==arr[i][j]){
                    a=i;b=j;
                }
            }
        }
        a++;
        b++;
        System.out.println(ret);
        System.out.println(a+" "+b);
    }
}